presentation/src/main/java/com/stop/ui/mission/MissionViewModel.kt

```diff
@@ -92,15 +107,49 @@ class MissionViewModel @Inject constructor(
     private fun getBusNowLocation() {
         viewModelScope.launch {
             while (TIME_TEST < 60) {
-                _busNowLocationInfo.value = getBusNowLocationUseCase.getBusNowLocation(BUS_540_ID)
-                Log.d("MissionViewModel","busNowLocationInfo ${_busNowLocationInfo.value}")
+                getBusNowLocationUseCase(TEST_BUS_540_ID).apply {
+                    _busNowLocationInfo.value = this
+                }
                 delay(5000)
                 TIME_TEST += 1
             }
 
         }
     }
 
+    private suspend fun getSubwayTrainNowLocation() = withContext(Dispatchers.Main) {
+        getSubwayTrainNowStationUseCase(TEST_TRAIN_NUMBER, TEST_SUBWAY_NUMER)
+    }
+
+
+    private suspend fun getNowStationLocation() = withContext(Dispatchers.Main) {
+        startSubwayStation = getSubwayTrainNowLocation().stationName
+        getNowStationLocationUseCase(
+            startSubwayStation,
+            personCurrentLocation.longitude,
+            personCurrentLocation.latitude
+        )
+    }
+
+    private fun getSubwayRoute() {
+        viewModelScope.launch {
+            val startLocation = getNowStationLocation()
+            getSubwayRouteUseCase(
+                RouteRequest(
+                    startLocation.longitude,
+                    startLocation.latitude,
+                    TEST_SUBWAY_LONG,
+                    TEST_SUBWAY_LAT
+                ),
+                TEST_SUBWAY_NUMER.toString() + LINE,
+                startSubwayStation.dropLast(1), //"역" 버리기
+                TEST_END_SUBWAY_STATION
+            ).apply {
+                _subwayRoute.value = this
+            }
```

Comment:
```suggestion
            this@MissionViewModel._subwayRoute.value = getSubwayRouteUseCase(
                RouteRequest(
                    startLocation.longitude,
                    startLocation.latitude,
                    TEST_SUBWAY_LONG,
                    TEST_SUBWAY_LAT
                ),
                TEST_SUBWAY_NUMER.toString() + LINE,
                startSubwayStation.dropLast(1), //"역" 버리기
                TEST_END_SUBWAY_STATION
            )
```

invoke 함수 결괏값을 바로 넣어줄 수 있어요.

Reply:
👍 👍 
https://github.com/boostcampwm-2022/android10-PlzStop/pull/58/commits/be4b1132882c601f3188140e2a35c909d9342ef4